docs/source/highlights.md

```diff
@@ -218,13 +218,13 @@ We always welcome contributions in forms of comments, suggestions, and code impl
 The generic patterns/modules identified from the research prototypes will be integrated into MONAI core functionality.
 
 ### COPLE-Net for COVID-19 Pneumonia Lesion Segmentation
-[A reimplementation](https://github.com/Project-MONAI/MONAI/tree/master/research/coplenet-pneumonia-lesion-segmentation) of the COPLE-Net originally proposed by:
+[A reimplementation](https://monai.io/research/coplenet-pneumonia-lesion-segmentation) of the COPLE-Net originally proposed by:
```

Comment:
I see a `re-direction` page for 2 secs, from monai.io to github repo.
So I think maybe it's better to use github link directly?
https://github.com/Project-MONAI/research-contributions/tree/master/coplenet-pneumonia-lesion-segmentation

Thanks.

Reply:
the idea is to use https://monai.io/research/coplenet-pneumonia-lesion-segmentation as a permanent link, not depending on the specific folder structure of `Project-MONAI/research-contributions`